BepInEx.Core/Bootstrap/TypeLoader.cs

```diff
@@ -126,9 +163,8 @@ public static AssemblyDefinition CecilResolveOnFailure(object sender, AssemblyNa
     ///     selector.
     /// </returns>
     public static Dictionary<string, List<T>> FindPluginTypes<T>(string directory,
```

Comment:
```
[Fatal  : Preloader] System.IO.DirectoryNotFoundException: Could not find a part of the path 'BepInEx\patchers-providers'.
```
You should probably make the folder if it doesn't exist

Reply:
Done, interestingly, it seems v6 wasn't correctly enforcing the patchers one (it was, but in the chainloader when the folder isn't used anymore by this point)